### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by clifton311**
Hi Sadie,

Can you perform a code review?

### CODE REVIEW

File: database/data.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+let interior_features = {
+   appliances: ['Dishwasher','Dryer', 'Garbage disposal', 'Refriderator', 'Washer'],
+   heating_cooling: ['forced air', 'fan', 'speeder air'],
```
Comment (thesadie) [inline database/data.js]: I noticed you are using snake case for your variable names, consider camel case

File: database/index.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  } else {
+    console.log("Connected to database");
+  }
+});
+
+//function to load mock data
+const loadData = (callback) => {
+  mockFeatures.forEach(house => {
+    let sql = 'INSERT into features (type, year_built, heating, cooling, parking, lot) VALUES (?,?,?,?,?,?)';
+    let params = [house.type,house.year_built,house.heating,house.cooling,house.parking,house.lot];
```
Comment (thesadie) [inline database/index.js]: consider adding spaces between items in array

File: database/data.js (hunk 1 above)
Comment (thesadie) [reply database/data.js]: nevermind if this is in your database

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+const data = require('../database/index');
+
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+app.use(express.static(__dirname+'/../client/dist'));
+
+
+app.get('/house/:id', (req, res) => {
+  //function to seed the mysql database with mock data for the house features
```
Comment (thesadie) [inline server/server.js]: does this seed the database? seems more like it queries the database, is the comment out of sync?

File: database/mockFeatures.json [hunk 4]
```diff
@@ -0,0 +1,136 @@
+
```
Comment (thesadie) [inline database/mockFeatures.json]: where is the script that created these?

File: test/testspec.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  user: 'root',
+  password: '',
+  database: 'zillow'
+});
+
+
+
+describe('Testing Database', () => {
+  it('should have the correct data', function (){
+    connection.query('SELECT * from features limit 2', (err, results, field) => {
```
Comment (thesadie) [inline test/testspec.js]: i noticed you include `field` in the function signature of your call back but never use it anywhere. consider removing

File: test/testspec.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+      console.log(results[0].type)
+      expect(results[0].id).to.equal(13);
+      expect(results[0].type).to.be.a('string');
```
Comment (thesadie) [inline test/testspec.js]: maybe test that the names of the columns returned match your expectations

File: database/index.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+db.connect((err) => {
+  if (err) {
+    console.error('error connecting: ' + err);
+  } else {
+    console.log("Connected to database");
+  }
+});
+
+//function to load mock data
```
Comment (thesadie) [inline database/index.js]: I think this function seeds your database. Where does it get called?

File: database/index.js (hunk 2 above)
Comment (clifton311) [reply database/index.js]: yeah, I should be doing that.

File: database/mockFeatures.json (hunk 4 above)
Comment (clifton311) [reply database/mockFeatures.json]: at first I was generating them by manually, but it was taking too long, so I used a an web program called mockaroo that helps generate mock data for you. I just downloaded the file from there.

File: server/server.js (hunk 3 above)
Comment (clifton311) [reply server/server.js]: this comment is out of place, let me take it out. You are right, the function queries the database.

File: test/testspec.js (hunk 6 above)
Comment (clifton311) [reply test/testspec.js]: that is great suggestion! yes, I know my tests are lacking : /.

File: database/mockFeatures.json (hunk 4 above)
Comment (thesadie) [reply database/mockFeatures.json]: i think they want us to write functions to generate the fake data, i'd check in with leslie

**Pull request closed without merging**